pkgs/data/themes/sweet-nova/default.nix

```
@@ -0,0 +1,51 @@
+{ fetchFromGitHub
+, gitUpdater
+, lib
+, stdenvNoCC
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "sweet-nova";
+  version = "unstable-2023-04-02";
+
+  src = fetchFromGitHub {
+    owner = "EliverLara";
+    repo = "Sweet";
+    rev = "8a5d5a7d975567b5ae101b9f9d436fb1db2d9b24";
+    hash = "sha256-FVcXBxcS5oFsvAUDcwit7EIfgIQznl8AYYxqQ797ddU=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+    cd kde
+    mkdir -p aurorae/themes
+    mv aurorae/Sweet-Dark aurorae/themes/Sweet-Dark
+    mv aurorae/Sweet-Dark-transparent aurorae/themes/Sweet-Dark-transparent
+    rm aurorae/.shade.svg
+    mv colorschemes color-schemes
+    mkdir -p plasma/look-and-feel
+    mv look-and-feel plasma/look-and-feel/com.github.eliverlara.sweet
+    mv sddm sddm-Sweet
+    mkdir -p sddm/themes
+    mv sddm-Sweet sddm/themes/Sweet
+    mv cursors icons
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -d $out/share
+    cp -r Kvantum aurorae color-schemes icons konsole plasma sddm $out/share
```

Usually `share` is populated by the name of application.

```suggestion
    install -d $out/share/sweet-nova/
    cp -r Kvantum aurorae color-schemes icons konsole plasma sddm $out/share/sweet-nova/
```

---

The applications would still be able to use it like the files were present in  `/usr/share/{Kvantum,aurorae,..}` ? My concern was that KDE would not be able to detect the theme using this path.

---

Update: I tried building it this way, which makes every piece of the theme unaccessible for KDE Plasma as I assumed earlier.

---

By looking at the already existent code for Sweet, it puts things at share/themes. Does it work?

---

This puts it in `share/themes` because its the location GTK themes are expected to be. Likewise, Kvantum is expected in `share/Kvantum` and so on.